### ISSUE TITLE
Imports.FromGlobalUsings does not support generic types

### ISSUE THREAD

**Issue by amcasey** [dotnet/roslyn#5485]
There is no global equivalent of ```using static System.Collections.Generic.List<int>;```.   It is not clear what would happen if the string "System.Collections.Generic.List<int>" were passed in - hopefully there would just be a binding failure and a diagnostic.

**Comment by amcasey** [comment]
It also doesn't de-dup target symbols, leading to incorrect ambiguities during lookup.  I'm going to try to fix that one since it's important for scripting.

**Comment by amcasey** [comment]
Ah, generics will be silently dropped:

```
if (!targetString.IsValidClrNamespaceName())
{
    continue;
}
```

Still, I think we should either forbid types entirely or support generics.

**Comment by gafter** [comment]
I am not sure if you are reporting one bug or two, and I am not sure what the symptoms are. Can you phrase this as an end-to-end scenario (e.g. Some program that improperly compiles or fails to compile)?

**Comment by amcasey** [comment]
The second issue - not de-duping - is in PR: https://github.com/dotnet/roslyn/pull/5506.

**Comment by amcasey** [comment]
Consider a library declaring a type
```C#
public class Helper<T>
{
    public static void Do(T t) { /* ... */ }
}
```

If you create a compilation with global using ```Helper<int>``` and the following source, it will fail to compile:
```C#
class C
{
    static void Main()
    {
        Do(1);
    }
}
```

**Comment by amcasey** [comment]
It's probably worth mentioning that it would fail even if we fixed this bug, since regular C# code ignores global usings.  But the submission ```> Do(1)``` is affected.

**Comment by jaredpar** [comment]
Is this a bug or not?  Do we allow bound generics in global usings?  If so why does @amcasey last sample fail to compile?  Intuitively it seems like it sould work.  

``` 
using static Helper<int>;

class C
{
    static void Main()
    {
        Do(1);
    }
}
```

**Comment by amcasey** [comment]
@jaredpar, the compiler currently ignores global usings entirely.  However, in interactive scenarios, where they are considered, they are strictly less powerful than "local" usings.  In particular, bound (or, AFAIK, unbound) generics are not supported.

**Comment by leppie** [comment]
This works though:
```
using H = F.Helper<int>;
namespace F {
  using static H;
  ...
}
```

Along the same lines, any reason the following should not work?
```
using H = Helper<int>;
using static H;
```

**Comment by jaredpar** [comment]
@amcasey okay.  Moving to interactive then if this is an interactive only scenario.
